transport/http/transport.go

```diff
@@ -183,6 +185,16 @@ func InnocenceWindow(d time.Duration) TransportOption {
 	}
 }
 
+// DialContext specifies the dial function for creating TCP connections on the
+// outbound.
+//
```

Comment:
Would be good to mention that the default dial context has a 30 second timeout and respects the KeepAlive option, whereas the override will not.